src/optimization/DAG.h

```diff
@@ -0,0 +1,34 @@
+//
+// Created by nomaddo on 17/12/18.
+//
+
+#ifndef VC4C_DAG_H
+#define VC4C_DAG_H
+
+#include "../Graph.h"
+#include "../intermediate/IntermediateInstruction.h"
+#include "../DebugGraph.h"
+
+using namespace vc4c;;
```

Comment:
Don't use a namespace globally in a header! Since the class `DAG` already lies in `vc4c`, there is no need to use the namespace here at all.